deploy/cluster-manager/config/samples/operator_open-cluster-management_clustermanagers.cr.yaml

```diff
@@ -13,3 +13,8 @@ spec:
     featureGates:
     - feature: DefaultClusterSet
       mode: Enable
+  # Enable ManifestWorkReplicaSet for e2e
+  workConfiguration:
```

Comment:
we should not by default enable it in the example

Reply:
okay, removed.